src/components/header/header.component.jsx

```diff
@@ -1,18 +1,19 @@
 import React from 'react';
 import {Link} from 'react-router-dom';
+import {Nav} from 'react-bootstrap';
 
 
 import '../header/header.style.css';
 
 const Header = () => {
 return (
-    <div className = 'header'>
-        <p className = " brand-title"> <Link to = '/' >Anuj</Link></p>
-        <ul className = 'header-items'>
-            <li><Link to = '/todo' > ToDo </Link></li>
-            <li><Link to = '/markdown' >MarkDown </Link></li>
-        </ul>
-    </div>
+  <Nav className = 'navbar header bg-dark pb-5 '>
+    <p className = " brand-title"> <Link to='/'>Anuj</Link></p>
+    <ul className = 'header-items text-white mr-auto'>
+      <li><Link to = '/todo' > Todo </Link></li>
+        <li><Link to = '/markdown' >Markdown </Link></li>
+      </ul>
```

Comment:
use proper indentation <ul> and </ul> should be on same vertical line and all <li> should be in same line.